src/constants.hpp

```diff
@@ -42,10 +42,21 @@
  */
 //#define DISABLE_SHUTDOWN_VIA_BUTTON
 
-/* uncomment the below line to enable the rotary encoder for volume setting (only for AiOplus)
- * um den Drehgeber zu unterstützen bitte in der nächste Zeile den Kommentar entfernen (nur für AioPlus)
+
+/* uncomment the below line to enable the rotary encoder for volume setting
+ * um den Drehgeber zu unterstützen bitte in der nächste Zeile den Kommentar entfernen
+ */
+// #define ROTARY_ENCODER
+
+/* uncomment the below line to enable the rotary encoder with long/short press like THREEBUTTON configuration
+ * um den Drehgeber mit lang/kurz Drehung wie 3-Knopf-Konfiguration zu benutzen bitte den Kommentar entferenen in der nächsten
+ * Zeile entfernen
  */
-//#define ROTARY_ENCODER
+// #define ROTARY_LONGPRESS
```

Comment:
@boerge1 With ROTARY_LONGPRESS the rotary encoder assumes both functions of the up/down buttons in the 3 button layout:
Short turn (1 click) -> up/down
Fast turn (2 or more clicks) -> upLong/downLong